src/nnsight/schema/format/types.py

```diff
@@ -110,13 +116,14 @@ def deserialize(self, handler: DeserializeHandler) -> Node:
             target=self.target.deserialize(handler),
             args=[try_deserialize(value, handler) for value in self.args],
             kwargs={
-                key: try_deserialize(value, handler) for key, value in self.kwargs.items()
+                key: try_deserialize(value, handler)
```

Comment:
I know it is slightly less efficient, but I think we should avoid adding formatting changes that don't effect functionality to PR's which are doing things changing functionality. Especially in scripts not containing any functional changes.

In an optimal setup, we'd already have a consistent formatting style, and each PR would have a check for linting. In our current circumstances, use `git add -patch {file}` to go through and first add functional changes, to push to one brach. Then commit the rest to a seperate branch and make a seperate PR. Or just routinely spend some time going through the code and making little cleanups and push

Reply:
wait nvm I see there is a flake8 linter in the checks